File: abc/abc279/a/solution_1.cpp
// https://atcoder.jp/contests/abc279/submissions/65772568
#include <bits/stdc++.h>
using namespace std;

int main() {
    string S;
    cin >> S;
    
    int ans = 0;
    for (char c : S) {
        if (c == 'w') ans += 2;
        else ans += 1;
    }
    cout << ans << "\n";
}

File: abc/abc326/c/solution_1.py
# https://atcoder.jp/contests/abc326/submissions/65805674
N, M = map(int,input().split())
A = list(map(int, input().split()))

from bisect import bisect_left

A.sort()
ans = 0
for i in range(N):
    r = A[i] + M
    tmp = bisect_left(A, r) - i
    ans = max(ans, tmp)
print(ans)
